### ISSUE TITLE
не совсем понял зачем нужно присвоение 2

### ISSUE THREAD

**Issue by OlegThomasLyasov** [Anastasia289/sem#1]
https://github.com/Anastasia289/sem/blob/65da3766e11ade32246b9b8e98690d8e051617a8/%D0%B4%D0%B7/Program.cs#L228

**Comment by Anastasia289** [comment]
Для наглядности, чтобы видеть куда поставили фигуру, их же несколько.  Сначала я еще присваивала единички тем клеткам, которые она пробивает. Просто чтобы видеть не уехали ли у меня строки\столбцы.
Или вопрос в том, почему именно 2?

**Comment by OlegThomasLyasov** [comment]
Да, вопрос почему именно 2. Может тогда лучше сделать значение true13.10.2022, 14:45, "Anastasia289" ***@***.***>:
Для наглядности, чтобы видеть куда поставили фигуру, их же несколько.  Сначала я еще присваивала единички тем клеткам, которые она пробивает. Просто чтобы видеть не уехали ли у меня строки\столбцы.
Или вопрос в том, почему именно 2?

**Comment by Anastasia289** [comment]
Может, и лучше. В изначальной версии я еще собиралась увеличивать эту двойку каждый раз когда она попадала под огонь, чтобы было видно какие фигуры в опасности. Потом решила, что это совсем уже лишнее, но числа остались.
